src/desktop_develop.js

```diff
@@ -240,7 +240,8 @@ class DesktopDevelopBuilder {
         // windows parses them for the version too. We don't really care: nobody
         // sees them. We just give the installer a static name, so you'll just
         // see this in the 'about' dialog.
-        const version = type.startsWith('win') ? '0.0.0-nightly.' + buildVersion : buildVersion;
+        // Turns out if you use 0.0.0 here it makes Squirrel crash, so we use 0.0.1.
```

Comment:
Just so we know which platform it crashed on:
```suggestion
        // Turns out if you use 0.0.0 here it makes Squirrel crash (particularly on Windows), so we use 0.0.1.
```

Reply:
Point. Added: https://github.com/vector-im/riot-builder/commit/39d4705175b150f6898d9f2d553490c7bc7dc1ed